libs/sdk-ui-filters/src/AttributeFilter/AttributeFilterButton.tsx

```diff
@@ -348,12 +348,33 @@ export const AttributeFilterButtonCore: React.FC<IAttributeFilterButtonProps> =
         [state.validOptions, state.offset, state.limit, resolvedParentFilters],
     );
 
+    const {
+        error: originalTotalCountError,
+        result: originalTotalCount,
+        status: originalTotalCountStatus,
+    } = useCancelablePromise<number>(
+        {
+            // eslint-disable-next-line
```

Comment:
Please let me know if there is any possible way to specify why I'm disabling eslint. 

The es-lint error was stating that there is exact duplicity for both `totalCount` cancelable promises.

Reply:
You can write another comment above this one to describe it